tensorflow/lite/kernels/fully_connected.cc

```diff
@@ -1135,21 +1146,22 @@ void FullyConnectedPerChannelInt16(const OpData* data,
   op_params.output_offset = output->params.zero_point;
   op_params.quantized_activation_min = data->output_activation_min;
   op_params.quantized_activation_max = data->output_activation_max;
-  if (bias && bias->type == kTfLiteInt64) {
+
+  if (data->quantized_bias_type == kTfLiteInt32) {
     reference_integer_ops::FullyConnectedPerChannel(
         op_params, data->per_channel_output_multiplier.data(),
         data->per_channel_output_shift.data(), GetTensorShape(input),
         GetTensorData<int16_t>(input), GetTensorShape(filter),
         GetTensorData<int8_t>(filter), GetTensorShape(bias),
-        GetTensorData<int64_t>(bias), GetTensorShape(output),
+        GetTensorData<int32_t>(bias), GetTensorShape(output),
```

Comment:
Comment from internal reviewer: "why do we use std::int32_t in the similar code in conv.cc?"